src/editor/validator.ts

```diff
@@ -650,4 +652,24 @@ export class Validator {
 
         return fuse.search(searchString);
     }
+
+    validateImports(stmts?: ImportStatement[]) {
+        if (!stmts) {
```

Comment:
Given a list of import statement validates all `Importable` objects in the AST against it.